src/page.h

```diff
@@ -876,11 +886,15 @@ class page
             string emission_blk_no   = std::to_string(current_values.blk_no - 1);
             string emission_coinbase = xmr_amount_to_str(current_values.coinbase, "{:0.3f}");
             string emission_fee      = xmr_amount_to_str(current_values.fee, "{:0.3f}");
+            string emission_coinbase_human = fmt::format("{:n}", static_cast<int64_t>(current_values.coinbase/1e9));
+            string emission_fee_human = fmt::format("{:n}", static_cast<int64_t>(current_values.fee/1e9));
```

Comment:
libfmt is supposed to be able to format a float/double with thousand separator and fixed decimal places, but I get a run time error. So, I am casting to an int here which means circulating supply on front end will be a whole number. The API endpoint is not affected by this and returns the correct fractional amount.